src/Me/Navigation/Navbar.tsx

```diff
@@ -50,12 +49,10 @@ const Navbar = () => {
         {isAdmin && (
           <MenuItem to="/me/admin" icon={IconMentors} label="Admin" />
         )}
-        <Link href={router.pathname.includes('/me') ? '/' : router.pathname}>
```

Comment:
Link redirects to homepage while the user is still not logged out.
No need. Logout redirects to auth0 which will redirect to homepage anyway.